build.gradle

```diff
@@ -31,19 +31,37 @@ repositories {
 
 checkstyle {
     maxWarnings = 0
-    toolVersion = '8.10.1'
     // need to set configDir to rootDir otherwise submodule will use submodule/config/checkstyle
     configDir = new File(rootDir, 'config/checkstyle')
 }
 
 pmd {
-    toolVersion = "6.44.0"
+    toolVersion = "6.50.0"
     ignoreFailures = true
     sourceSets = [sourceSets.main, sourceSets.test]
     reportsDir = file("$project.buildDir/reports/pmd")
     ruleSetFiles = files("config/pmd/ruleset.xml")
 }
 
+// https://jeremylong.github.io/DependencyCheck/dependency-check-gradle/configuration.html
+dependencyCheck {
+    // Specifies if the build should be failed if a CVSS score above a specified level is identified.
+    // range of 7-10 fails the build, anything lower and it doesn't fail the build
+    failBuildOnCVSS = 7.0
```

Comment:
Shouldnt this by a system property like the others ?

Reply:
No, that is through Jenkins, this is using GitHub actions, so there is no need